src/components/main/Main.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Main } from './Main';
+import { messages as mainMessages } from './messages';
+import { messages as searchMessages } from './components/search-section/messages';
+
+describe('Main component', () => {
+  describe('Main component - search and loading interactions', () => {
```

Comment:
```suggestion
  describe('Search and loading interactions', () => {
```
если хочешь объединить. но тоже как будто не очень понятно почему именно для Error button надо тесты отдельно выносить